### DISCUSSION TITLE
Indian Market/Exchange/Brokers Support

### DISCUSSION THREAD

**Discussion by DaemonStark** [blankly-finance/blankly#211] [Q&A]
Hi, 
Is there any plan to integrate Indian Market/Exchanges in this? there are various brokers in India, and it would be great if we could use this to integrate with it for addressing wider use cases.

I'll try to browse through the code to understand this, and will try something out of this sort to further contribute to this project.

Thank you.

**Comment by EmersonDove** [comment]
Hey we don't currently have any plans for that but a few people have asked so it might be something that we should look into. 

If you're interested in contributing, you just have to create a relatively short interface file (for example here is alpaca's): https://github.com/blankly-finance/blankly/blob/main/blankly/exchanges/interfaces/alpaca/alpaca_interface.py)

And then use this template so it can be used anywhere: https://github.com/blankly-finance/blankly/blob/main/blankly/exchanges/interfaces/alpaca/alpaca.py

and you will get full backtesting, paper trading, live trading on the exchange.

**Reply by DaemonStark** [reply]
Sure,
Thank you.
